hibernate-reactive-core/src/test/java/org/hibernate/reactive/MutinySessionTest.java

```diff
@@ -389,7 +394,7 @@ public void reactiveUpdateVersion(TestContext context) {
 								.call(session::flush)
 								.eventually(session::close)
 						)
-						.chain( () -> selectNameFromId( 5 ) )
+						.replaceWith( selectNameFromId(5) )
```

Comment:
Only because this is a PR about the style of the code:
shouldn't it be `.replaceWith( selectNameFromId( 5 ) )` (with spaces surrounding the 5)?

Reply:
I don't think there's any reason on earth to put spaces around a numeric literal. It actually makes it *harder* to parse the nested parens, not *easier*.